src/main/kotlin/org/rust/cargo/toolchain/CargoCommandLine.kt

```diff
@@ -16,4 +17,21 @@ data class CargoCommandLine(
     val workingDirectory: Path? = null,
     val environmentVariables: EnvironmentVariablesData = EnvironmentVariablesData.DEFAULT,
     val nocapture: Boolean = true
-)
+) {
+    companion object {
+        fun forProject(
```

Comment:
Found a better solution for this :) 

https://github.com/intellij-rust/intellij-rust/commit/559218c81dcce533b5ebc09d4615663f019ad93c#diff-96ed2d1e44784b834de08e4c720d9f9bL21